libraries/AP_ICEngine/AP_ICEngine.cpp

```diff
@@ -134,8 +134,8 @@ const AP_Param::GroupInfo AP_ICEngine::var_info[] = {
 
     // @Param: OPTIONS
     // @DisplayName: ICE options
-    // @Description: Options for ICE control. The DisableIgnitionRCFailsafe option will cause the ignition to be set off on any R/C failsafe.
-    // @Bitmask: 0:DisableIgnitionRCFailsafe,1:DisableRedineGovernor
+    // @Description: Options for ICE control. The DisableIgnitionRCFailsafe option will cause the ignition to be set off on any R/C failsafe. If ThrottleWhileDisarmed is set then throttle control will be allowed while disarmed for planes when in MANUAL mode.
```

Comment:
Is it only manual still? Or some others also?

Reply:
its only manual as the non-manual code paths zero throttle before this point in the plane directory.